### DISCUSSION TITLE
where to find sleek version?

### DISCUSSION THREAD

**Discussion by aisbergde** [ransome1/sleek#674] [Q&A] [answered]
I installed sleek from the Windows "Microsoft Store", and I can't find any version information in sleek. There is no "about" section or something similar. There is also no information in the "settings". How to know the installed version? Why not just make this information available in the program?
![image](https://github.com/ransome1/sleek/assets/9020187/9c8439ec-4528-40f9-8538-8cff1ec6717c)

**Comment by ransome1** [comment, marked as answer]
@aisbergde if you press the `alt` key, the window menu will appear. Here you can find the the `About` entry, which will show the current version.

**Reply by aisbergde** [reply]
Or at least toogle this behavior in settings. Or place an information in settings.
Then a user:
- will know about the existence of a menu (how he could know otherwise?)
- can decide, if he wants to see the menu by default or not. For example, I normally prefer to see menus.

**Comment by aisbergde** [comment]
Good to know! But how a user should know? Maybe it would be better to show the menu by default?

**Comment by ransome1** [comment]
It used to be the default until users complaint about expressing it uglyfies the UI. A solution would be to let users decide if they want to see it or not using a setting.

**Reply by aisbergde** [reply]
Yes, this would be the best (settings). Otherwise, it can happen to uses what happened to me :-)

**Comment by patbarron** [comment]
I was just about to open an issue requesting an "About" dialog with the version number, when I came across this thread...

Unfortunately, the current implementation (pressing the "alt" key) makes this practically impossible to find, unless you already know it's there.  I had no idea about the "alt" key popping up a menu bar until I saw this thread....
